tests/rules.1.yaml

```diff
@@ -34,3 +34,21 @@ tests:
       periodic: 5
       until: test_traffic.complete
       after: deploy
+- name: test run_action
```

Comment:
This change breaks test_rules.py. It's a small fix: you just need to change the length assertion from 2 to 3.